node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -138,7 +138,9 @@ func (ms *P2PMessageService) setupDht(bootPeers []string) {
 	ctx := context.Background()
 	var options []dht.Option
 	options = append(options, dht.BucketSize(20))
-	options = append(options, dht.Mode(dht.ModeServer)) // allows other peers to connect to this node
+	options = append(options, dht.Mode(dht.ModeServer))                                               // allows other peers to connect to this node
+	options = append(options, dht.ProtocolPrefix(DHT_PROTOCOL_PREFIX))                                // need this to allow custom NamespacedValidator
+	options = append(options, dht.NamespacedValidator("scaddr", stateChannelAddrToPeerIDValidator{})) // all records prefixed with /scaddr/ will use this custom validator
```

Comment:
`scaddr` is a namespace within the DHT, so it is at a lower level than the `DHT_PROTOCOL_PREFIX`. A record in the `scaddr` namespace will look like this (assuming `DHT_PROTOCOL_PREFIX = "/nitro"`: `/nitro/scaddr/0x12345...9089`

Reply:
Sorry, I misunderstood what you were saying. We want to remove the hardcoded "scaddr" on that line and use a `const` instead:
```
const (
	DHT_RECORD_PREFIX                             = "/" + DHT_NAMESPACE + "/"
	DHT_NAMESPACE                                    = "scaddr"
	DHT_PROTOCOL_PREFIX    protocol.ID = "/nitro" // use /nitro/kad/1.0.0 instead of /ipfs/kad/1.0.0
)
	

options = append(options, dht.ProtocolPrefix(DHT_PROTOCOL_PREFIX))                                     // need this to allow custom NamespacedValidator
	options = append(options, dht.NamespacedValidator(DHT_NAMESPACE, stateChannelAddrToPeerIDValidator{})) //
```